Content.Server/Chemistry/EntitySystems/RehydratableSystem.cs

```diff
@@ -30,7 +32,10 @@ private void Expand(EntityUid uid, RehydratableComponent comp)
     {
         _popups.PopupEntity(Loc.GetString("rehydratable-component-expands-message", ("owner", uid)), uid);
 
-        var target = Spawn(comp.TargetPrototype, Transform(uid).Coordinates);
+        var randomMob = _random.Next(0, comp.TargetPrototype.Count);
+
+        var target = Spawn(comp.TargetPrototype[randomMob], Transform(uid).Coordinates);
```

Comment:
```suggestion
        var randomMob = _random.Pick(comp.TargetPrototype);

        var target = Spawn(randomMob, Transform(uid).Coordinates);
```

Reply:
also i dont think TargetPrototype is a good name when its multiple possible targets

Reply:
I replaced the randomize code with the one EmoGarbage404 proposed and renamed TargetPrototype to PossibleSpawns